source/features/user-local-time.tsx

```diff
@@ -162,49 +165,7 @@ function init(signal: AbortSignal): void {
 	observe(selector, insertUserLocalTime, {signal});
 }
 
-async function profileInit(): Promise<void> {
-	const login = getCleanPathname();
-	if (login === getUsername()) {
-		return;
-	}
-
-	const datePromise = getLastCommitDate(login);
-	const race = await Promise.race([delay(300), datePromise]);
-	if (race === false) {
-		// The timezone was undeterminable and this resolved "immediately" (or was cached), so don't add the icon at all
-		return;
-	}
-
-	attachElement('.vcard-details', {
-		append: () => createTimeElement(datePromise),
-	});
-}
-
-// TODO: Plz replace with JSX-less Preact or sumthin 🥺
-// Passing `datePromise` around is we-ird
-function createTimeElement(datePromise: Promise<string | false>): JSX.Element {
-	const placeholder = <span className="v-align-middle">Guessing local time…</span>;
-	const container = (
-		<li className="vcard-detail pt-1 css-truncate css-truncate-target">
-			<ClockIcon/> {placeholder}
-		</li>
-	);
-
-	void display({datePromise, placeholder, container});
-	return container;
```

Comment:
@kidonng I reverted https://github.com/refined-github/refined-github/pull/5779 because the _temporary_ duplicate code ended up being permanent and the feature was unmaintainable as is.